packages/vscode-extension/messages.json

```diff
@@ -66,15 +68,15 @@
 	"EXTENSION_PROJECT_STARTUP_MESSAGE_PROJECT_NEEDS_SETUP_ACCOUNT": "No account has been set up for this project. Click \"SuiteCloud: Setup Account\" to link your project with your account.",
 
 	"IMPORT_FILES_ERROR_IMPORT_TO_SUITEAPP_NOT_ALLOWED": "Files cannot be imported to a SuiteApp project. You can only import files to Account Customization Projects.",
-	"IMPORT_FILES_FINISHED": "The importing process has finished.",
-	"IMPORT_FILES_IMPORTING_FILE": "Importing file...",
+	"IMPORT_FILES_ERROR_NOT_ALLOWED_FOLDER": " Files can only be imported from \"/FileCabinet/SuiteScripts\", \"/FileCabinet/Templates/E-mail Templates\", \"/FileCabinet/Templates/Marketing Templates\" and \"/FileCabinet/Web Site Hosting Files\" folders in an Account Customization Project.",
```

Comment:
imho it sounds better if we rephrase it like: "In an Account Customization Project files can only be imported from..."

Reply:
This message still needs to be reviewed by @Montsant19 . We'll take into consideration your comment.

Reply:
I don't think using this adverbial of place at the beginning sounds natural in English. We will review it again and let you know, @antoniofruci. Thanks!

Reply:
New message:
**Files in an Account Customization Project can only be imported from the following File Cabinet folders: "/SuiteScripts", "/Templates/E-mail Templates", "/Templates/Marketing Templates" and "/Web Site Hosting Files".**